### ISSUE TITLE
[CATA][Talents][Shaman] Totemic Wrath (Elemental)

### ISSUE THREAD

**Issue by thunderbolt94** [NRG-Wow/NRGWow-Bugtracker#139]
Reporter(Player/Character) Name: Djeljana

Description: Totemic Wrath (Talent) will not overwrite Flametounge totem

Expected behaviour: Totemic Wrath (Talent) Should give its Buffs of 10% of spellpower when you put down Flametounge totem

Steps to reproduce the problem: 
1.Put Down Flametounge totem
2.It will appear only Flametounge totem buff (6%) instead of Totemic Wrath (10%) spellpower bonus

Here is Reference at 0:55 he put Flametounge totem and he get buff Totemic wrath from the elemental talent tree, and this is from retail >
https://www.youtube.com/watch?v=XpSzD2T985c

**Comment by Wilcalaf** [comment]
Need to verify if its even a bug. if you a reference please provide
in any case, there is no reason to use Flametongue totem when you have that talent. the whole purpose of that talent is to let you use another fire totem.
which of them is supposed to be dominant? requires research.
here you have a post saying that it did get overrided by flametongue on retail
https://us.battle.net/forums/en/wow/topic/2065567176

if you have a valid reference please provide it so you can help us with the research.
thanks

**Comment by Illusie** [comment]
it does overwritten on atlantiss.
if you use flametongue totem you get only Totemic Wrath.

**Comment by Bourne185** [comment]
Totemic Wrath does not affect the Flametongue Totem.

Here are my references.

https://wow.gamepedia.com/Totemic_Wrath
"Totemic Wrath applies to ALL Fire Totems, and renders  [Flametongue Totem] utterly useless."

Also,

http://wowwiki.wikia.com/wiki/Flametongue_Totem
"Its effect is inferior to Totem of Wrath, and doesn't stack with it."

I other words, Totemic Wrath is not designed to affect Flametongue totem, and is only meant to buff the other totems. If you choose to use the Flametongue totem, thats up to you.

**Comment by thunderbolt94** [comment]
As you posted there, totemic wrath affects All fire totems, that include
flametounge totem, not stacking with it, and when im a healer i dont want
to use 1 min fire totem buff

сре, 8. авг 2018. 23.30 Bourne185 <notifications@github.com> је написао/ла:

**Comment by Bourne185** [comment]
And as i also stated, Totemic wrath renders flametongue useless and it doesnt stack 

as in the buffs are not compatible, so the totemic wrath will not affect the flametongue totem at all.

**Comment by Bourne185** [comment]
Also, you want totemic wrath to give flametongue totem 10%, guess what, it already is :D 

flame tongue wont work as long as u have that talent (hence why i said it doesnt stack)

**Comment by thunderbolt94** [comment]
No,i just checked when you put Flametounge totem it gives 6 % of spell power not 10 %, so its not affected  by the Totemic Wrath, Flametounge totem needs to be replaced with totemic wrath talent. It gives you 5 min Totemic wrath buff (10%) spell power

**Comment by Bourne185** [comment]
I am telling you that the two effects do not stack. use a different totem. The flametonque totem does not activate totemic wrath. What i am saying is that you need to use a different totem to get the buff. Look at my posts, all of them say it renders flametongue useless

**Comment by Bourne185** [comment]
The affects cannot overwrite eachother, because they are the same effect.

**Comment by Bourne185** [comment]
try it on  [Searing Totem],  [Magma Totem] or  [Fire Elemental Totem]

**Comment by thunderbolt94** [comment]
They give, i know that, but we are speaking of Flametounge totem, i say how it should work, not saying that you need to use it, noone force you to use Flametounge totem but i m saying it Should give the totemic wrath buff,

**Comment by Bourne185** [comment]
but all the references and research i have done disagrees with that statement. Due to the fact that they are the same effect, and that the effects clearly say they do not stack ( which means combine) one must take over the other. So when you place the totem, it is using its proper stat, not the talent stat. They do not combine.

**Comment by Babunibre** [comment]
"Totemic Wrath applies to ALL Fire Totems, and renders  [Flametongue Totem] utterly useless. Other Shaman can safely drop  [Searing Totem],  [Magma Totem] or  [Fire Elemental Totem] depending on the situation if this is present in the group. In addition, Totemic Wrath does not stack with a Demonology Warlock's  [Demonic Pact]."
As you said in your reference link,this is pretty reliable, first in the sentence you should see "ALL fire totems", which means all of the totems will get the talent buff(totemic Wrath).
But than I think you are badly apprehending the meaning "utterly useless", which should mean you will get totemic wrath anyway so why would you in right mind use flametongue > wont give you anything since talent gives you the best. But it doesn't mean it wont / shouldn't give you totemic wrath. As stated its just useless or pointless to put it.

**Comment by Illusie** [comment]
https://github.com/Atlantiss/BugTracker/issues/9197
taken from atlantiss I hope it might help.

**Comment by thunderbolt94** [comment]
http://www.wowhead.com/forums&topic=185555/totems-buffs-and-you

Fire
Flametongue
Same buff as a mage's Arcane Brilliance. Inferior to and overwritten by an elemental shaman's Totemic Wrath or a demo lock's Demonic Pact.

DPS fire totem
improved personal dps
Bottom line:
Elemental shaman can drop a dps totem to their heart's content, totemic wrath will provide the superior 10% SP buff. Resto will have to verify that the SP buff is covered before deciding which totem to drop, if you have an ele shaman, mage or demo lock skip Flamtongue and drop a dps totem. Enhance Shaman suffer a noticeable single target dps loss if they do not drop Searing Totem so I would not worry about the buff

**Comment by thunderbolt94** [comment]
Heres the proof

**Comment by Illusie** [comment]
If I can mention #109 here would be nice :)

**Comment by thunderbolt94** [comment]
https://imgur.com/a/VTNW0fp
 
Here Are the pictures

**Comment by Illusie** [comment]
I dont see any problem.
You know what try watch some Dragon Soul videos from retail
try here : https://www.youtube.com/user/BloodLegionIllidan/videos

If you find in buff list Flametongue 6% + Totemic Wrath 10% (16% overall) screenshot and post.

**Comment by thunderbolt94** [comment]
Man, again you dont understand, i sayed 10 times , its shouldnt stack, i posted up there, But it should be replaced with totemic wrath

**Comment by thunderbolt94** [comment]
It doesnt metter when you put totem, it should give you anytime buff from totemic wrath, i just want to help the server to be better (best) and i will post any bug i find to help,

**Comment by Babunibre** [comment]
I think you Illusie are still confused mate, this is not intended disccusion of totemic wrath being overwritten by some other flametongue buff from other Restoshaman or somebody else/whatsoever. Why he posted this bugg for is very simple: as Elemental shaman you should be able to put flametongue totem and gain Totemic Wrath talent nevertheless even if it render other specific attributes of flametongue totem to nothing. It has nothing to do with anything else(so far any other bugg with auras being replaced or whatsoever hasn't been detected). 
Simple thing: you are lazy to put searing totem every 1 minute, so you can put flametongue for 5min and just get basic 10% buff from it and not even one more thing as it render useless flametongue buff(so totem itself is useless since talent exists).

**Comment by Illusie** [comment]
Oh you talking only about Ele spec (the talent) you probably right.
Flametongue is mostly used for Resto shamans to provide them 6% SP in Arenas

As Elemental spec with the talent you always should get Totemic Wrath.
According to my knowledge stronger Aura always overwritten weaker aura. hf

**Comment by thunderbolt94** [comment]
Im Glad you understand now, xD  HF

**Comment by Illusie** [comment]
my pleasure. I deleted older messages because I thought we talk about buff stacking in raid situation.

from atlantiss:
https://imgur.com/a/GTSpgHv

I will still remain the link of my Buff stacking bug report to here #109 

have nice day.
